backend/migrate/migrations/scripts/migration_7.sql

```diff
@@ -0,0 +1,6 @@
+DROP VIEW view_hijacks;
+
+ALTER TABLE hijacks
+    ALTER COLUMN type TYPE VARCHAR ( 5 ) USING replace(replace(replace(replace(type::VARCHAR ( 1 ), 'S', 'S|-|-'), '0', 'E|0|-'), '1', 'E|1|-'), 'Q', 'Q|0|-')::VARCHAR ( 5 );
+
+CREATE OR REPLACE VIEW view_hijacks AS SELECT key, type, prefix, hijack_as, num_peers_seen, num_asns_inf, time_started, time_ended, time_last, mitigation_started, time_detected, timestamp_of_config, under_mitigation, resolved, active, ignored, configured_prefix, comment, seen, withdrawn, outdated, peers_withdrawn, peers_seen FROM hijacks;
```

Comment:
is there any change in the view and you had to replace it?

Reply:
The migrator complained if someone tries to alter the table while there is already a dependent view. So I apply the strategy we applied in other contexts (drop and reinstate the view).

Reply:
is it the same though? if it's not we need to change graphql

Reply:
Does graphql look into lengths of column fields? If not, then it is the same (I have not changed any names or anything).

Reply:
nope